pkg/startup/v8/derive.go

```diff
@@ -62,11 +62,11 @@ func (derivedType) KubeReserved(cs *api.OpenShiftManagedCluster, role api.AgentP
 }
 
 func (derivedType) MasterCloudProviderConf(cs *api.OpenShiftManagedCluster) ([]byte, error) {
-	return derivedpkg.MasterCloudProviderConf(cs, true)
+	return derivedpkg.MasterCloudProviderConf(cs, false)
```

Comment:
Is this plan to turn this on later in v8 once the setting is actually available?

Reply:
yes :) as soon as we get errata released and tested